src/flamenco/vm/fd_vm_jitproto.c

```diff
@@ -0,0 +1,1259 @@
+/*
```

Comment:
Would you be able to put all of the JIT-related code in a subdirectory? So it's clear to auditors and contributors what code is meant for production and what code is experimental. Thanks so much 🙌

Reply:
Good idea. Done